ydb/library/yql/providers/yt/lib/res_pull/res_or_pull.cpp

```diff
@@ -133,12 +144,34 @@ bool TSkiffExecuteResOrPull::WriteNext(const NYT::TNode& item) {
         }
     }
 
+    // For now this method is used only for the nodes that are lists of columns
```

Comment:
Could you please add some tests for the newly found cases to the yql agent tests later?

Reply:
Sure thing! I will do it together with YDB commit promotion